base-template/about-base-template.md

```diff
@@ -11,84 +24,349 @@ The base template goes one step further, in that it helps you create any templat
+
+Note:
+
+* Many microcontent fields need to be maintained manually which adds a maintenance burden to a project.
+* If any of the microcontent on a site is inaccurate or dated, it will reduce the trustworthiness of all the microcontent on the site, as the user typically won’t be able to determine which of the content is trustworthy.
+* As such, a project should only adopt microcontent fields which are automatically maintained by your documentation publishing system, or where there is an established, low-effort process to ensure the microcontent is maintained.
+
+## Primary goal
+
+{Template author tip: Concisely state the primary purpose of this doctype, ideally in one short sentence.}
+
+## Target audience and user stories
+
+{Template author tip: You should document the key user stories that this template is likely to be used for.}
+
+{Docset owner tip: Select the user stories which are applicable to your project.}
+
+This {doctype} template aims to address the following user stories:
+
+* As a {user persona}, I want to {achieve a specific goal}, so that I can {achieve a larger business objective}.
+
+{Template author tip: You may optionally add a “docset owner tip” explaining when the following user story should or should not be included.}
+
+* As a {user persona}, I want to {achieve a specific goal}, so that I can {achieve a larger business objective}.
+    * {Docset owner tip: Include this user story when … .}
+
+## Nature of content
+
+{Template author tip: 
+
+* Discuss the nature of the content for this doctype.
+* Consider how it will impact maintenance.
+* Is this content expressed best with multi-media, such as diagrams, screenshots or video?
+* If so, reference your style guide which should set recommendations on image and screenshot size, video length, style etc.
+
+}
+
+## Priorities for each quality criteria
+
+{Template author tip: 
+
+* Each doctype addresses different use cases. It target’s different user personas, with different needs.
+* As such, there are different weightings applied to quality characteristics for each doctype. Reference documentation needs to comprehensively address all the features in the latest version, but can tolerate lower quality writing. Tutorials need only cover a sample of use cases of an older version of the software, but there is a greater need for writing quality.}
+
+{Docset owner tip: Select the quality criteria which will be important for your project.}
+
+The following table lists the relative importance of the following quality criteria for {doctype} documents.
+
+
+|Quality criteria|Priority to address|
+|--- |--- |
+|Currency: Alignment with the latest version of software.|{Select one of: Must, Should, Nice to have (May), Not Applicable (N/A), Should not}|
+|Currency: Alignment with a specified version of software.||
+|Comprehensive: Covers every feature and use case.||
+|Well written: Unambiguous.||
+|Well written: Concise, high signal to noise ratio, high information density.||
```

Comment:
Dunno what "high signal to noise ratio" means.

Reply:
Yeh, good catch. I'll delete reference to it.
It is a term used in relation to old school radio or TV signals. A geeky engineering concept.
Good signal, low static means you get a clear audio or picture.
Low signal means that you mostly get fuzzy static.